db.go

```diff
@@ -75,7 +71,7 @@ func checkConnection(db *DB) {
 		var one uint
 		err := db.QueryRow("SELECT 1").Scan(&one)
 		if err != nil {
-			log.Fatal("Database connection failed:", err)
+			log.Fatal("Database connection failed: ", err)
```

Comment:
Should this be a Fatalf? How does log.Fatal handle two args?

Reply:
https://github.com/mozilla-services/tigerblood/pull/102/commits/ffbb9e9c570b77a265704d95e454aaa59ccfa809